plugins/inputs/influxdb_v2_listener/influxdb_v2_listener.go

```diff
@@ -38,7 +38,7 @@ const (
 	defaultWriteTimeout = 10 * time.Second
 )
 
-var ErrEOF = errors.New("EOF")
+var ErrEOF = io.EOF
```

Comment:
Could we replace the `ErrEOF` in the code?

Reply:
Done.